CRM/Event/Form/ParticipantFeeSelection.php

```diff
@@ -495,11 +495,6 @@ private function formatFieldsForOptionFull(): void {
               $optionFullTotalAmount += $option['amount'] ?? 0;
             }
           }
-          else {
-            if (!empty($defaultPricefieldIds) && in_array($optId, $defaultPricefieldIds)) {
-              unset($optionFullIds[$optId]);
-            }
-          }
```

Comment:
This comes from [point 2.2 here](https://issues.civicrm.org/jira/browse/CRM-15364), but not adding the form rule when we aren't on the front end below covers this so it is no longer needed.